src/uu/touch/src/touch.rs

```diff
@@ -238,10 +238,14 @@ fn parse_date(str: &str) -> FileTime {
     // be any simple specification for what format this parameter allows and I'm
     // not about to implement GNU parse_datetime.
     // http://git.savannah.gnu.org/gitweb/?p=gnulib.git;a=blob_plain;f=lib/parse-datetime.y
-    match time::strptime(str, "%c") {
-        Ok(tm) => local_tm_to_filetime(to_local(tm)),
-        Err(e) => panic!("Unable to parse date\n{}", e),
+    let formats = vec!["%c", "%F"];
+    for f in formats {
+        match time::strptime(str, f) {
+            Ok(tm) => return local_tm_to_filetime(to_local(tm)),
+            _ => (),
+        }
     }
+    panic!("Unable to parse date\n");
```

Comment:
please add a test to trigger this error (and probably use show_error instead of a panic)

Reply:
Test added and use `show_error`.